chromium_src/components/permissions/permission_context_base.cc

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "components/permissions/permission_context_base.h"
+
+#define PermissionContextBase PermissionContextBase_ChromiumImpl
+#include "../../../../components/permissions/permission_context_base.cc"
+#undef PermissionContextBase
+
+#include "brave/components/permissions/permission_lifetime_manager.h"
```

Comment:
we need DEPS for `chromium_src` dirs as well

Reply:
@iefremov we do have them so this must not be violating any existing rules

Reply:
@iefremov it's handled with this: https://github.com/brave/brave-core/blob/73f5e6f7d2ddd3051507f401dc55b134fb749b47/chromium_src/components/permissions/DEPS#L3